CHANGELOG.md

```diff
@@ -84,7 +84,7 @@
 
 ### Added
 
-- The entire library is now compatible with the [Sass module system](https://sass-lang.com/blog/the-module-system-is-launched), and backwards compatibile with the existing Sass import system. This means you can import  the whole library or its parts with `@use`/`@forward` or `@import`, and override variables in all of the available ways. The functions and mixins are all available in both systems
+- The entire library is now compatible with the [Sass module system](https://sass-lang.com/blog/the-module-system-is-launched), and backwards compatibile with the existing Sass import system. This means you can import the whole library or its parts with `@use`/`@forward` or `@import`, and override variables in all of the available ways. The functions and mixins are all available in both systems
```

Comment:
How did you run prettier on those `.md` doc files? Does the project's `lint` task automatically include those? Otherwise we should add a custom task for it to prevent them from going off again in the future 🤔 

I can take care of the extra setup if we don't have it. Just assign me an issue.

Reply:
Yes, I’ll add that to the package.json & CI 🙏